hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -574,3 +574,35 @@ wait_for_root_certificate(){
   echo "Timed out waiting on $count root certificates. Current timestamp " $(date +"%T")
   return 1
 }
+
+execute_s3a_tests() {
```

Comment:
I thought this function could be used to test against non-docker cluster, but since `testlib.sh` is already specific to docker-based tests, it does not make much sense.

Moved.